.github/workflows/recreate-runner.yml

```diff
@@ -0,0 +1,41 @@
+name: Recreate Linux self-hosted runner on schedule
+
+# Uncomment and use this code below once this is ready:
+# on:
+#   workflow_dispatch:
+#   schedule:
+#     # Once each 24 hours, at 1 during the night
+#     - cron: "0 0 1 1/1 *"
+
+on: pull_request
+
+jobs:
+  recreate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          version: 12
+      - run: npm install @octokit/action
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: Wait for idle runner
+        id: killable
+        env:
+          GITHUB_TOKEN: ${{ secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN }}
```

Comment:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Can we try this?

Reply:
Already tested before, here is the error you'll get from oktokit:
```
2021-05-05T17:56:09.6378791Z (node:1615) UnhandledPromiseRejectionWarning: HttpError: Resource not accessible by integration
2021-05-05T17:56:09.6381523Z     at /home/runner/work/homebrew-core/homebrew-core/node_modules/@octokit/request/dist-node/index.js:68:23
2021-05-05T17:56:09.6383038Z     at processTicksAndRejections (internal/process/task_queues.js:97:5)
2021-05-05T17:56:09.6385217Z     at async is_runner_running (/home/runner/work/homebrew-core/homebrew-core/.github/actions/wait-for-idle-runner.js:10:20)
2021-05-05T17:56:09.6391184Z (node:1615) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
2021-05-05T17:56:09.6395902Z (node:1615) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

We need the `HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN` scope for this to work.